fault/actions.py

```diff
@@ -36,9 +36,6 @@ def __init__(self, port, value):
 
 class Print(Action):
     def __init__(self, port, format_str="%x"):
-        if port.isoutput():
```

Comment:
Does this work for magma, i.e. does `simulator.get_value(port)` work for input ports? I can add a test for this.

Reply:
Added test. It works.